packages/web-components/src/components/va-statement-of-truth/va-statement-of-truth.scss

```diff
@@ -4,32 +4,32 @@
 @import '~@department-of-veterans-affairs/css-library/dist/tokens/scss/variables.scss';
 
 :host :not(h3) {
-  font-size: 1.06 * $v3-font-base-size;
+  font-size: var(--vads-font-size-source-sans-normalized);
 }
 
 :host h3 {
   font-family: var(--font-serif);
-  margin: 24px 0;
+  margin: var(--units-3) 0;
 }
 
 :host > p {
-  margin: 24px;
+  margin: var(--units-3);
 }
 
 @media screen and (max-width: $small-screen) {
   :host .short-line {
-    width: var(--vads-size-line-length-1);
+    text-wrap: pretty;
   }
```

Comment:
> [!note]
> Needed to change this because this width declaration caused the text to extend beyond the viewport when the fonts were set to "very large". I believe that this replacement matches the intention of the width rule, but improves the flexibility with user agent customizations.